docs/source/serialization.mdx

```diff
@@ -50,257 +61,374 @@ Ready-made configurations include the following models:
+config = AutoConfig.from_pretrained("distilbert-base-uncased")
+onnx_config = DistilBertOnnxConfig(config)
+```
 
-</Tip>
+The resulting object has several useful properties. For example you can view the
+ONNX operator set that will be used during the export:
+
+```python
+# Returns 11
+print(onnx_config.default_onnx_opset)
+```
 
-### Quantization
+You can also view the outputs associated with the model as follows:
 
-ONNX exporter supports generating a quantized version of the model to allow efficient inference.
+```python
+print(onnx_config.outputs)
+```
 
-Quantization works by converting the memory representation of the parameters in the neural network to a compact integer
-format. By default, weights of a neural network are stored as single-precision float (*float32*) which can express a
-wide-range of floating-point numbers with decent precision. These properties are especially interesting at training
-where you want fine-grained representation.
+which gives:
+
+```python
+OrderedDict([("last_hidden_state", {0: "batch", 1: "sequence"})])
+```
 
-On the other hand, after the training phase, it has been shown one can greatly reduce the range and the precision of
-*float32* numbers without changing the performances of the neural network.
+Notice that the outputs property follows the same structure as the inputs; it
+returns an `OrderedDict` of named outputs and their shapes. The output structure
+is linked to the choice of feature that the configuration is initialised with.
+By default, the ONNX configuration is initialized with the `default` feature
+that corresponds to exporting a model loaded with the `AutoModel` class. If you
+want to export a different model topology, just provide a different feature to
+the `task` argument when you initialize the ONNX configuration. For example, if
+we wished to export DistilBERT with a sequence classification head, we could
+use:
 
-More technically, *float32* parameters are converted to a type requiring fewer bits to represent each number, thus
-reducing the overall size of the model. Here, we are enabling *float32* mapping to *int8* values (a non-floating,
-single byte, number representation) according to the following formula:
+```python
+from transformers import AutoConfig
 
-$$y_{float32} = scale * x_{int8} - zero\_point$$
+config = AutoConfig.from_pretrained("distilbert-base-uncased")
+onnx_config_for_seq_clf = DistilBertOnnxConfig(config, task="sequence-classification")
+# Returns OrderedDict([('logits', {0: 'batch'})])
+print(onnx_config_for_seq_clf.outputs)
+```
 
 <Tip>
 
-The quantization process will infer the parameter *scale* and *zero_point* from the neural network parameters
+All of the base properties and methods associated with
+[`OnnxConfig`](https://github.com/huggingface/transformers/blob/c4fa908fa98c3d538462c537d29b7613dd71306e/src/transformers/onnx/config.py#L52)
```

Comment:
```suggestion
[`OnnxConfig`]
```
Let the documentation resolve the link to this class maybe? Or did you really need the source code here?

Reply:
Yeah, I added the permalink because I wasn't sure if the doc will resolve the link. But you're right, I'll remove the URL for now :)

Reply:
It's actually not in the API documentation so you should either add it or keep the link to GitHub :-/

Reply:
Hehe, OK I'll re-instate the link for now and add the API documentation in a separate PR to keep this one focused on just the user guide.